modules/@shopify/checkout-sheet-kit/ios/AcceleratedCheckoutButtons+Extensions.swift

```diff
@@ -31,14 +31,11 @@ import SwiftUI
 @available(iOS 16.0, *)
 extension PayWithApplePayButtonLabel {
     static func from(_ string: String?, fallback: PayWithApplePayButtonLabel = .plain) -> PayWithApplePayButtonLabel {
-        guard let string else {
+        guard let string, let value = map[string] else {
             return fallback
         }
-        return map[string] ?? .plain
-    }
 
-    public static func == (lhs: PayWithApplePayButtonLabel, rhs: PayWithApplePayButtonLabel) -> Bool {
-        return String(describing: lhs) == String(describing: rhs)
```

Comment:
Swift 6.2 makes the `PayWithApplePayButtonLabel` struct conform to `Equatable`, so this is no longer needed